pkgs/games/tome2/default.nix

```diff
@@ -0,0 +1,46 @@
+{ stdenv, fetchFromGitHub,  makeDesktopItem, ncurses, libX11, boost, cmake }:
+
+let
+  pname = "tome2";
+  desc = "A dungeon crawler similar to Angband, based on the works of Tolkien.";
```

Comment:
Call this `description`, and then on line 42 you can just say `inherit description`.